### PR TITLE
Add affiliations and ORCIDs to AUTHORS file

### PR DESCRIPTION
**Pull request by leouieda**
The authorship guidelines introduced in fatiando/contributing#13 specify
that authors on Zenodo archives will be taken from the AUTHORS.md file
and need to have full names, affiliation, and ORCIDs (optionally)
included in that file.

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If adding new functionality, add an example to the docstring, gallery, and/or tutorials.

### CODE REVIEW

**Comment by leouieda** [comment]
@Xarthisius @andersy005  @remram44 @hugovk @matthewturk @jrleeman @danshapero you have all made contributions to Pooch so I'd like to include you in the Zenodo archives. Some of you have already been added in the past but not all are in the AUTHORS.md file. **Would it be OK to add your information and include you as authors on the Zenodo archives of future releases?** If you agree, I can get your information from the JOSS paper.

@Xarthisius if you agree could you please provide your full name, affiliation, and ORCID?

**Comment by remram44** [comment]
I agree :heavy_check_mark:

**Comment by Xarthisius** [comment]
> @Xarthisius if you agree could you please provide your full name, affiliation, and ORCID?

Sure and thank you!

```
* [Kacper Kowalik](https://github.com/Xarthisius) - National Center for Supercomputing Applications, Univeristy of Illinois at Urbana-Champaign, USA (ORCID: 0000-0003-1709-3744)
```

would it make sense to use links for ORCID?

```
(ORCID: [https://orcid.org/0000-0003-1709-3744](0000-0003-1709-3744))
```

**Comment by leouieda** [comment]
Thanks, Kacper. Yeah, the links are nicer.

File: AUTHORS.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
 * [Anderson Banihirwe](https://github.com/andersy005)
-* [Hugo van Kemenade](https://github.com/hugovk)
+* [Hugo van Kemenade](https://github.com/hugovk) - Independent (Non-affiliated) (ORCID: [0000-0001-5715-8632](https://www.orcid.org/0000-0001-5715-8632))
+* [Kacper Kowalik](https://github.com/Xarthisius) - National Center for Supercomputing Applications, Univeristy of Illinois at Urbana-Champaign, USA (ORCID: [0000-0003-1709-3744](https://www.orcid.org/0000-0003-1709-3744))
 * [John Leeman](https://github.com/jrleeman)
-* [Santiago R. Soler](https://github.com/santisoler)
-* [Leonardo Uieda](http://www.leouieda.com/)
+* [Rémi Rampin](https://github.com/remram44) - New York University, USA (ORCID: [0000-0002-0524-2282](https://www.orcid.org/0000-0002-0524-2282))
+* [Santiago Soler](https://github.com/santisoler) - Universidad Nacional de San Juan, Argentina (ORCID: [0000-0001-9202-5317](https://www.orcid.org/0000-0001-9202-5317))
```
Comment (santisoler) [inline AUTHORS.md]:
```suggestion
* [Santiago Soler](https://github.com/santisoler) - CONICET, Argentina; Instituto Geofísico Sismológico Volponi, Universidad Nacional de San Juan, Argentina (ORCID: [0000-0001-9202-5317](https://www.orcid.org/0000-0001-9202-5317))
```

Comment (leouieda) [reply AUTHORS.md]: Sorry, I always forget that one. Could you add it to the other projects as well?

Comment (santisoler) [reply AUTHORS.md]: Yes, no problem!

**Comment by andersy005** [comment]
> Would it be OK to add your information and include you as authors on the Zenodo archives of future releases?

Sure! 

```
* [Anderson Banihirwe](https://github.com/andersy005) - The US National Center for Atmospheric Research, USA (ORCID: 0000-0001-6583-571X)
```

**Comment by leouieda** [comment]
@matthewturk @jrleeman @danshapero I'll merge this PR soon to make a new release with some of the new things we've added. If you're happy being added to the file, let me know (just a :+1: is fine). Otherwise, feel free to add yourselves at any time :slightly_smiling_face:

**Comment by matthewturk** [comment]
Oh!  I am sorry I did not see this.  I would be very keen to be added, but understand if it's too late now!  I'm ORCID 0000-0002-5294-0198.

**Comment by santisoler** [comment]
Right on time @matthewturk! Feel free to edit you information if it's wrong or something is missing.

File: AUTHORS.md [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
+* [Kacper Kowalik](https://github.com/Xarthisius) - National Center for Supercomputing Applications, Univeristy of Illinois at Urbana-Champaign, USA (ORCID: [0000-0003-1709-3744](https://www.orcid.org/0000-0003-1709-3744))
 * [John Leeman](https://github.com/jrleeman)
-* [Santiago R. Soler](https://github.com/santisoler)
-* [Leonardo Uieda](http://www.leouieda.com/)
+* [Matthew Turk](https://github.com/matthewturk) - University of Illinois, USA (ORCID: [0000-0002-5294-0198](https://www.orcid.org/0000-0002-5294-0198))
+* [Rémi Rampin](https://github.com/remram44) - New York University, USA (ORCID: [0000-0002-0524-2282](https://www.orcid.org/0000-0002-0524-2282))
+* [Santiago Soler](https://github.com/santisoler) - CONICET, Argentina; Instituto Geofísico Sismológico Volponi, Universidad Nacional de San Juan, Argentina (ORCID: [0000-0001-9202-5317](https://www.orcid.org/0000-0001-9202-5317))
+* [Leonardo Uieda](https://github.com/leouieda) - University of Liverpool, UK (ORCID: [0000-0001-6123-9515](https://www.orcid.org/0000-0001-6123-9515))
```
Comment (leouieda) [inline AUTHORS.md]:
```suggestion
* [Rémi Rampin](https://github.com/remram44) - New York University, USA (ORCID: [0000-0002-0524-2282](https://www.orcid.org/0000-0002-0524-2282))
* [Santiago Soler](https://github.com/santisoler) - CONICET, Argentina; Instituto Geofísico Sismológico Volponi, Universidad Nacional de San Juan, Argentina (ORCID: [0000-0001-9202-5317](https://www.orcid.org/0000-0001-9202-5317))
* [Matthew Turk](https://github.com/matthewturk) - University of Illinois at Urbana-Champaign, USA (ORCID: [0000-0002-5294-0198](https://www.orcid.org/0000-0002-5294-0198))
* [Leonardo Uieda](https://github.com/leouieda) - University of Liverpool, UK (ORCID: [0000-0001-6123-9515](https://www.orcid.org/0000-0001-6123-9515))
```

**Comment by leouieda** [comment]
No worries @matthewturk :slightly_smiling_face: 

@santisoler thanks for adding the info. I updated the affiliation to match the JOSS paper and moved Matt to the bottom with the rest of the cool kids (alphabetical by last name) :sunglasses:

**Comment by matthewturk** [comment]
👍

**Pull request merged**